test/admission_test.go

```diff
@@ -77,35 +82,30 @@ func TestAdmissionControl(t *testing.T) {
 				}
 			}
 
-			readers := 50
-			readers_replication := 3
-			read_entries := 1000 / readers
-			writers := 500
-			writers_replication := 10
-			write_entries := 1000 / writers
-			wg.Add(readers*readers_replication + writers*writers_replication)
+			readers := writeLimit * 4
+			read_entries := 20
+
+			writers := writeLimit * 4
+			write_entries := 20
+			wg.Add(readers + writers)
```

Comment:
This `wg.Add` doesn't feel more or less strongly associated with the readers or the writers, so I think there should be a newline here, there shouldn't be one between `read_entries` and `writers`